collections-lists/ConvertIAsyncEnumerableToList/ConvertIAsyncEnumerableToListTests/IAsyncEnumerableToListTests.cs

```diff
@@ -0,0 +1,46 @@
+namespace ConvertIAsyncEnumerableToListTests;
+
+public class IAsyncEnumerableToListTests
+{
+    [Fact]
+    public async Task GivenGetUsersAsListAsync_WhenCalled_ThenReturnsListOfUsers()
+    {
+        // Act
+        var usersList = await AsyncUserListConverter
+            .GetUsersAsListAsync();
```

Comment:
(3 of 3) And then pass in the much lower value from the tests, so they run as fast as possible.

```suggestion
            .GetUsersAsListAsync(10);
```